driver.py

```diff
@@ -149,11 +149,13 @@ def collect_experience(self, record=1, vis=0, n_steps=None, noise_flag=True, sta
             if not noise_flag:
                 do_keep_prob = 1.
             
-            a, a_probs = self.sess.run(fetches=[alg.action_test, alg.action_probs], feed_dict={alg.states: np.reshape(observation, [1, -1]),
+            a, a_means = self.sess.run(fetches=[alg.action_test, alg.action_means], feed_dict={alg.states: np.reshape(observation, [1, -1]),
                                                                     alg.do_keep_prob: do_keep_prob,
                                                                     alg.noise: noise_flag,
                                                                     alg.temp: self.env.temp})
 
+            a_probs = common.compute_action_probs(a, a_means[0], np.repeat(self.env.sigma, a.shape[0]))
```

Comment:
don't we need to use all the a_means instead of a_means[0]? 

The action taken is a vector [a,b,c], each with their own mean and log_std right?

Reply:
I think this the issue here was that the a_means was returned as 2d array, (1, actions_len)